spring-cloud-stream-core-docs/src/main/asciidoc/index.adoc

```diff
@@ -1,6 +1,6 @@
 [[spring-cloud-stream-reference]]
 = Spring Cloud Stream Reference Guide
-Sabby Anandan; Marius Bogoevici; Eric Bottard; Mark Fisher; Ilayaperumal Gopinathan; Gunnar Hillert; Mark Pollack; Patrick Peralta; Glenn Renfro; Thomas Risberg; Dave Syer; David Turanski; Janne Valkealahti; Benjamin Klein; Vinicius Carvalho
+Sabby Anandan; Marius Bogoevici; Eric Bottard; Mark Fisher; Ilayaperumal Gopinathan; Gunnar Hillert; Mark Pollack; Patrick Peralta; Glenn Renfro; Thomas Risberg; Dave Syer; David Turanski; Janne Valkealahti; Benjamin Klein; Vinicius Carvalho; Oleg Zhurakousky
 :doctype: book
```

Comment:
Me too 😄

Reply:
The author list from the release train project takes precedence in the final aggregated doc. So you need to add your names there. Here is the file: https://raw.githubusercontent.com/spring-cloud/spring-cloud-stream-starters/master/spring-cloud-stream-docs/src/main/asciidoc/index.adoc.